mcodingbot/tasks/stats.py

```diff
@@ -0,0 +1,136 @@
+from __future__ import annotations
+
+import asyncio
+import logging
+from dataclasses import dataclass
+from math import log
+from typing import TYPE_CHECKING, Any
+
+from mcodingbot.config import CONFIG
+
+if TYPE_CHECKING:
+    from mcodingbot.bot import Bot
+
+
+LOGGER = logging.getLogger(__file__)
+
+
+async def loop_update_channels(bot: Bot) -> None:
+    while True:
+        try:
+            await update_channels(bot)
+        except Exception:
+            LOGGER.error("Failed to update channel stats:", exc_info=True)
+
+        await asyncio.sleep(5 * 60)
```

Comment:
shouldn't this be updated too since `update_channels` no longer throw assertion errors?

Reply:
It will still raise errors if it's missing permissions to edit channels, or if discord's api is down, etc.

Reply:
oh i see